normandy/recipes/models.py

```diff
@@ -149,12 +170,16 @@ def compute_implementation_hash(self):
         return hashlib.sha1(self.implementation.encode()).hexdigest()
 
     def save(self, *args, **kwargs):
-        # Save first so the upload is available.
-        super().save(*args, **kwargs)
```

Comment:
Actions aren't uploaded anymore, they are included in-line in the API, so this wasn't doing anything except getting in the way.